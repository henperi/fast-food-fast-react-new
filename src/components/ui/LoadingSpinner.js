import React from "react";

export const LoadingSpinner = () => (
  <section className="m-t-40 text-center">
    <h2 className="setup-page hide t-40">
      Loading... <i className="fa fa-spin fa-spinner" />{" "}
    </h2>
  </section>
);
