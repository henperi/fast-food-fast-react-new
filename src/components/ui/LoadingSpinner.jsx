import React from 'react';

export default () => (
  <section className="m-t-40 text-center">
    <h2 className="setup-page hide t-40">
      Loading...
      <i className="fa fa-spin fa-spinner" />
    </h2>
  </section>
);
