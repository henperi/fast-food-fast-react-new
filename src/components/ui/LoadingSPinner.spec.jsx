import React from 'react';
import { shallow } from 'enzyme';
import LoadingSpinner from './LoadingSpinner';

test('should render LoadingSpinner correctly', () => {
  const wrapper = shallow(<LoadingSpinner />);
  expect(wrapper).toMatchSnapshot();
});
