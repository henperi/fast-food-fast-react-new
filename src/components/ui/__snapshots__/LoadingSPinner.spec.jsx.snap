// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingSpinner correctly 1`] = `
<section
  className="m-t-40 text-center"
>
  <h2
    className="setup-page hide t-40"
  >
    Loading...
    <i
      className="fa fa-spin fa-spinner"
    />
  </h2>
</section>
`;
